### DISCUSSION TITLE
[CSV Import], Docker, unclear issue feedback.

### DISCUSSION THREAD

**Discussion by myhobby2** [firefly-iii/firefly-iii#5152] [Firefly III]
Thanks a lot for the effort in creating Firefly3. Everything seems to work in a docker environment and I trying to get the import to recognize the details I would like to see for my bank accounts. I can confirm that the import works but during the fine-tuning of the setup for an account I ran into the following message. Flushed, reboot both docker instances but it is not clear what is happening exactly. I constantly using the same bankdetails.csv file to recheck the import results. 

Line #0: Submission HTTP error: Status code is 500: {"message":"Internal Firefly III Exception: Attempt to read property \"transaction_group_id\" on null","exception":"ErrorException"}.

![image](https://user-images.githubusercontent.com/49870972/135635582-58718e79-19ae-4287-864c-dfbaa1f7cfc8.png)

![image](https://user-images.githubusercontent.com/49870972/135635847-40459694-4fd8-4eb8-be46-e8415f4d4892.png)

![image](https://user-images.githubusercontent.com/49870972/135635902-4d6c6d6d-043f-430e-be6f-550044c1c481.png)

**Comment by JC5** [comment]
You ran into this bug: https://github.com/firefly-iii/firefly-iii/issues/5145

Firefly III also verifies duplicate transactions across **deleted** transactions. However, instead of reporting the duplicate, Firefly III crashes.

I hope to release 5.6.2 soon that will remedy this.

**Reply by myhobby2** [reply]
Thanks for clarifying ! Will wait for the next release to have this resolved
